docs/content/integrations/airflow.mdx

```diff
@@ -1,24 +1,25 @@
 ---
-title: Dagster with Airflow | Dagster
+title: "Dagster + Airflow | Dagster Docs"
 description: The dagster-airflow package allows you to import Airflow DAGs into Dagster jobs and assets, as well as trigger job runs from Airflow.
 ---
 
 # Using Dagster with Airflow
 
 <CodeReferenceLink filePath="examples/with_airflow" />
 
-The [`dagster-airflow`](/\_apidocs/libraries/dagster-airflow) library provides interoperability between Dagster and Airflow. The main scenarios for using the Dagster Airflow integration are:
+The [`dagster-airflow`](/\_apidocs/libraries/dagster-airflow) library provides interoperability between Dagster and Airflow. Using `dagster-airflow`, you can:
 
-- You want to do a lift-and-shift migration of all your existing Airflow DAGs into Dagster jobs/Software-defined Assets (SDAs)
-- You want to trigger Dagster job runs from Airflow
+- Incorporate an Airflow Directed Acyclic Graph (DAG) directory as a Dagster repository, allowing you to do a lift-and-shift migration of your existing Airflow DAGs into Dagster jobs or Software-defined Assets (SDAs)
+- Orchestrate Dagster jobs from Airflow
+- Ingest DAGs from Airflow
 
 This integration is designed to help support users who have existing Airflow usage and are interested in exploring Dagster.
 
 ---
 
 ## Migrating to Dagster
 
-Interested in migrating from Airflow to Dagster? Check out the [migration guide](/integrations/airflow/migrating-to-dagster) for a step-by-step walkthrough.
+Interested in migrating from Airflow to Dagster? Check out the [migration guide](/integrations/airflow/migrating-to-dagster) for a step-by-step walkthrough. You can also refer to the [Airflow reference](/integrations/airflow/reference) for specific examples.
```

Comment:
If I understand it correctly, I don't think the reference provides specific migration examples (i.e. it's possible to use the integration without doing a migration).